src/Kinematics.cxx

```diff
@@ -6,26 +6,37 @@
  */
 
 #include "Kinematics.h"
-
+#include <pybind11/stl.h>
 ClassImp(Kinematics)
 
 Kinematics::Kinematics(
     Double_t enEleBeam, /*GeV*/
     Double_t enIonBeam, /*GeV*/
-    Double_t crossAng /*mrad*/
+    Double_t crossAng /*mrad*/    
     )
 {
-
+  srand(time(NULL));
+  // working version (slow):
+  efnpackage = py::module_::import("testEFlowimport");
+  pfnimport = efnpackage.attr("eflowPredict");
+  
+  //tensorflow = py::module_::import("tensorflow");
+  //keras = tensorflow.attr("keras");
+  //efnpackage = py::module_::import("energyflow");
+  //pfnimport = efnpackage.attr("archs");
+  //numpy = py::module_::import("numpy");
+  //model = keras.attr("models").attr("load_model")(modelname);
+  
   // set ion mass
   IonMass = ProtonMass();
-
+  
   // revise crossing angle
   crossAng *= 1e-3; // mrad -> rad
   crossAng = -1*TMath::Abs(crossAng); // take -1*abs(crossAng) to enforce the correct sign
 
   // TEST SETTINGS - for debugging calculations ///////////////
   // set main frame, used for calculations where there is ambiguity which frame is the correct frame to use
-  mainFrame = fHeadOn; // fLab, fHeadOn
+  mainFrame = fLab; // fLab, fHeadOn
```

Comment:
Why do you want to change this?

Reply:
Wanted to change this so that the HFStree I produced would be in lab frame by default, but I could just move the lines in Kinematics.cxx where I store this information for the tree.